src/main/java/com/ts/tserp/model/User.java

```diff
@@ -1,10 +1,70 @@
 package com.ts.tserp.model;
 
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
 
-@Entity
+@Entity(name="user")
 public class User {
 
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
 	private int id;
+	private String first_name;
+	private String last_name;
+	private int phone_no;
+	private String email;
+	private String role;
+	private String password;
+	public int getId() {
+		return id;
+	}
+	public void setId(int id) {
+		this.id = id;
+	}
+	public String getFirst_name() {
+		return first_name;
+	}
+	public void setFirst_name(String first_name) {
+		this.first_name = first_name;
+	}
+	public String getLast_name() {
+		return last_name;
+	}
+	public void setLast_name(String last_name) {
+		this.last_name = last_name;
+	}
+	public int getPhone_no() {
+		return phone_no;
+	}
+	public void setPhone_no(int phone_no) {
+		this.phone_no = phone_no;
+	}
+	public String getEmail() {
+		return email;
+	}
+	public void setEmail(String email) {
+		this.email = email;
+	}
+	public String getRole() {
+		return role;
+	}
+	public void setRole(String role) {
+		this.role = role;
+	}
+	public String getPassword() {
+		return password;
+	}
+	public void setPassword(String password) {
```

Comment:
Add regular expression to evaluate password. Password must contain 8 character atleast. Password should contain at least one capital character, 1 digit and 1 special character